.azure/applications/resource-registry-syncer-job/main.bicep

```diff
@@ -0,0 +1,83 @@
+targetScope = 'resourceGroup'
```

Comment:
What is needed when a new job is added now is pretty much a copy/paste of this bicep-file with the bicepparams with minor renaming + addition of the job name in the deployment-yaml. Will try to simplify much further moving forward